code/game/gamemodes/cult/cult_items.dm

```diff
@@ -374,4 +371,96 @@
 	suit = /obj/item/clothing/suit/cultrobesghost
 	shoes = /obj/item/clothing/shoes/cult/ghost
 	head = /obj/item/clothing/head/hooded/culthood/alt/ghost
-	r_hand = /obj/item/melee/cultblade/ghost
\ No newline at end of file
+	r_hand = /obj/item/melee/cultblade/ghost
+
+/obj/item/shield/mirror
+	name = "mirror shield"
+	desc = "An infamous shield used by Nar'Sien sects to confuse and disorient their enemies. Its edges are weighted for use as a throwing weapon - capable of disabling multiple foes with preternatural accuracy."
+	icon = 'icons/obj/cult.dmi'
+	icon_state = "mirror_shield"
+	item_state = "mirror_shield"
+	force = 5
+	throwforce = 15
+	throw_speed = 1
+	throw_range = 4
+	w_class = WEIGHT_CLASS_BULKY
+	attack_verb = list("bumped", "prodded")
+	hitsound = 'sound/weapons/smash.ogg'
+	var/illusions = 2
+
+/obj/item/shield/mirror/hit_reaction(mob/living/carbon/human/owner, atom/movable/hitby, attack_text = "the attack", final_block_chance = 0, damage = 0, attack_type = MELEE_ATTACK)
+	if(iscultist(owner))
+		if(istype(hitby, /obj/item/projectile))
+			var/obj/item/projectile/P = hitby
+			if(P.damage_type == BRUTE || P.damage_type == BURN)
+				if(P.damage >= 30)
+					var/turf/T = get_turf(owner)
+					T.visible_message("<span class='warning'>The sheer force from [P] shatters the mirror shield!</span>")
+					new /obj/effect/temp_visual/cult/sparks(T)
+					playsound(T, 'sound/effects/glassbr3.ogg', 100)
+					owner.Weaken(2)
+					qdel(src)
+					return FALSE

```

Comment:
I don't think this point will ever be hit and instead it will return . (which is null). Can be wrong though